src/SlippiGame.js

```diff
@@ -162,9 +162,12 @@ export default class SlippiGame {
       return this.gameEnd;
     }
 
-    // Trigger getFrames because that is where the flag is set
-    this.getFrames();
-    return this.gameEnd || null;
+    const slpfile = openSlpFile(this.input);
+
+    this.gameEnd = getGameEnd(slpfile);
```

Comment:
Since getGameEnd can now return `{}` which is truthy, if this function is ever called on an incomplete file, and then later called when the file is complete, it will always return `{}`. Might be best to return null to avoid this?

I may also write a test that simulates this because this is a commonly overlooked case.

Reply:
I just pushed a test that should fail until this is fixed.

Reply:
I forgot {} is truthy lol